### ISSUE TITLE
Trained models for 'person' label and/or ports

### ISSUE THREAD

**Issue by iitzco** [msracver/Flow-Guided-Feature-Aggregation#35]
The provided models were trained on the ImageNet VID dataset, which does not include a 'person' label. **Does anyone know where can I find a trained model for persons?**

Also, does anyone know of a Tensorflow/Pytorch implementation of this architecture? I'm having too much trouble making it run.

**Comment by manmanCover** [comment]
same problem...

**Comment by joe660** [comment]
> 提供的模型在不包含“人”标签的ImageNet VID数据集上进行了训练。**有谁知道我在哪里可以找到经过训练的人模型？**
> 
> 另外，有人知道这种架构的Tensorflow / Pytorch实现吗？我在使其运行时遇到了太多麻烦。

有pytorch版本吗？可以加好友交流一下这个模型吗

**Comment by Sanku-Yogesh** [comment]
yes here is the pytorch version https://github.com/Scalsol/mega.pytorch/issues/73#issue-754884469
